jupyter-js-widgets-nbextension/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jupyter-js-widgets-nbextension",
+  "version": "0.0.1",
+  "description": "Jupyter Widgets nbextension",
+  "main": "src/extension.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "prepublish": "npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipython/ipywidgets.git"
+  },
+  "author": "Jupyter Project",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/ipython/ipywidgets/issues"
+  },
+  "homepage": "https://github.com/ipython/ipywidgets#readme",
+  "dependencies": {
+    "backbone": "^1.2.3",
+    "html2canvas": "^0.5.0-beta4",
+    "jquery": "^2.2.0",
+    "jupyter-js-widgets": "file:///Users/jon/github.com/ipython/ipywidgets/jupyter-js-widgets",
```

Comment:
That's a mistake, thanks for catching it.  When you install using a rel path via the CLI, I did "npm install --save file:../..", apparently it will expand the path before saving.